kibiutils.js

```diff
@@ -33,6 +35,25 @@
       }
     };
 
+    var slugifyId = function (id) {
+      if (id == null) return;
+
+      var trans = {
+        '/' : '-slash-',
+        '\\?' : '-questionmark-',
+        '\\&' : '-ampersand-',
+        '=' : '-equal-',
+        '#' : '-hash-'
+      };
+      _.each(trans, function (val, key) {
+        var regex = new RegExp(key, 'g');
+        id = id.replace(regex, val);
+      });
+      id = id.replace(/[\s]+/g, '-');
```

Comment:
Remove the lodash dependency please 
Depend on the whole lodash to do 1 single loop seems a little too much

instead do 
```
for (var key in trans) {
  if (trans.hasOwnProperty(key)) {
      var regex = new RegExp(key, 'g');
      id = id.replace(regex, trans[key]); 
  }
}
```